go.mod

```diff
@@ -163,6 +164,7 @@ require (
 
 replace (
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210622111912-ef00f8ac3d76
+	github.com/evmos/ethermint => github.com/notional-labs/ethermint v0.6.1-0.20220825205659-53da09caeb52
```

Comment:
If I am not around to finalize this PR, basically just remove this replace and import an ethermint v0.19.1